.gitignore

```diff
@@ -38,3 +38,6 @@ dist/
 # project dirs
 /datasets
 /models
+
+# docker compose files
+/docker/.env
```

Comment:
looks like it is no longer used. could you remove it?

Reply:
Done. Thanks for reminding me.